service/src/main/java/uk/gov/hmcts/reform/fpl/model/configuration/DocumentUploadedNotificationConfiguration.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.fpl.model.configuration;
+
+import lombok.Builder;
+import lombok.Getter;
+import uk.gov.hmcts.reform.fpl.enums.cfv.ConfidentialLevel;
+import uk.gov.hmcts.reform.fpl.service.cafcass.CafcassRequestEmailContentProvider;
+
+/**
+ * Confidential level configuration for document uploaded notification
+ * e.g. sendToCafcassEngland
+ * If set to LA, only send notification to cafcass england when a LA level or non-confidential level document is
+ * uploaded. If null, no notification will be sent to cafcass england.
+ */
+@Getter
+@Builder(toBuilder = true)
+public class DocumentUploadedNotificationConfiguration {
+    private ConfidentialLevel sendToCafcassEngland;
+    private ConfidentialLevel sendToCafcassWelsh;
+
+    private ConfidentialLevel sendToRespondentSolicitor;
+    private ConfidentialLevel sendToChildSolicitor;
+
+    private ConfidentialLevel sendToDesignatedLA;
+    private ConfidentialLevel sendToSecondaryLA;
+    private ConfidentialLevel sendToLegalRepresentative;
+
+    private ConfidentialLevel sendToJudge;
+    private ConfidentialLevel sendToCTSC;
+
+    private ConfidentialLevel sendToCafcassRepresentative;
+
+    private ConfidentialLevel sendToOthersSelected;
+
+    private ConfidentialLevel sendToTranslationTeam;
+
+    private CafcassRequestEmailContentProvider cafcassRequestEmailContentProvider;
+
+    public static final DocumentUploadedNotificationConfiguration DEFAULT_NOTIFICATION_CONFIG =
+        DocumentUploadedNotificationConfiguration.builder()
+            .sendToCafcassEngland(ConfidentialLevel.LA)
```

Comment:
Does that mean only documents uploded by LA or solicitor will notify CAFCASS England?

Reply:
It means only non-confidential document and confidential document uploaded by LA will be send, confidential document uploaded by CTSC won't be sent.

But, I reviewed the existing logic. Looks like CTSC level hearing document should be sent to CAFCASS, while CTSC-level non-hearing document shouldn't.
I updated the configuration of hearing document.